helloworldwebapp/cleanup_app_and_pipelines.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+bold() {
+  echo ". $(tput bold)" "$*" "$(tput sgr0)";
+}
+
+source ~/spinnaker-for-gcp/scripts/install/properties
+
+~/spinnaker-for-gcp/scripts/manage/check_project_mismatch.sh
+
+read -p ". $(tput bold)You are about to delete all resources from the helloworldwebapp application and pipelines. This step is not reversible. Do you wish to continue (Y/n)? $(tput sgr0)" yn
```

Comment:
It renders as if `Y` is the default, but if I just hit enter nothing happens.

Reply:
done, handled default